src/generic/clipboard/paste-component/components/PasteButton.jsx

```diff
@@ -1,36 +0,0 @@
-import PropsTypes from 'prop-types';
-import { useParams } from 'react-router-dom';
-import { Button } from '@openedx/paragon';
-import { ContentCopy as ContentCopyIcon } from '@openedx/paragon/icons';
-
-const PasteButton = ({ onClick, text, className }) => {
-  const { blockId } = useParams();
-
-  const handlePasteXBlockComponent = () => {
-    onClick({ stagedContent: 'clipboard', parentLocator: blockId });
-  };
-
```

Comment:
We were receiving an `onClick` event of type `() => void`, so passing these parameters was unnecessary.